### ISSUE TITLE
Accordion a11y issues

### ISSUE THREAD

**Issue by nicoxxie** [IBM/carbon-components-react#1600]
## Detailed description

> Describe in detail the issue you're having.

Issues reported by DAP

IBM CI 162 Checkpoint Violation: 2.1.1 Keyboard
IBM Guidance: https://www.ibm.com/able/guidelines/ci162/keyboard.html
Test Tool Type: DAP

> Is this issue related to a specific component?

Widget with role of identifier must have no more than one tabbable element.?
--
  |   |   |   | Widget with role of tablist must have no more than one tabbable element.
  |   |   |   | Widget with role of tab must have no more than one tabbable element.
  |   |   |   | Widget with role of listbox must have no more than one tabbable element.
  |   |   |   | Widget with role of combobox must have no more than one tabbable element.
  |   |   |   | Widget with role of tab must have no more than one tabbable element.

> What did you expect to happen? What happened instead? What would you like to see changed?

Following DAP guidelines

> What browser are you working in?

Chrome

> What version of the Carbon Design System are you using?

6.55.5

> What offering/product do you work on? Any pressing ship or release dates we should be aware of?

IBM cloud

## Steps to reproduce the issue

1. Step one
2. Step two
3. Step three
4. etc.

> Please create a reduced test case in code sandbox
> https://codesandbox.io/s/r4zxq9pvq

## Additional information

- Screenshots or code
- Notes

**Comment by asudoh** [comment]
Couldn't reproduce the errors listed in the issue description from my quick run of DAP August rule. Feel free to re-open if you see otherwise, thanks!
